src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -154,8 +276,64 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityProps> = ({
         .times(isAsymAssetSide || isAsymRuneSide ? 2 : 1)
         .toFixed(),
     )
+    setCrytoAssetBalance(underlyingAssetAmountCryptoPrecision)
+    setFiatAssetBalance(underlyingAssetValueFiatUserCurrency)
+    setCrytoRuneBalance(underlyingRuneAmountCryptoPrecision)
+    setFiatRuneBalance(underlyingRuneValueFiatUserCurrency)
```

Comment:
As mentioned above, these should be removed as they're simply re-setting properties from `userLpData`

Reply:
https://github.com/shapeshift/web/pull/6175/commits/ce65270269aca06920532f2bb741e003db7495dd